src/words/EditButton.js

```diff
@@ -48,15 +48,24 @@ export default function EditButton({
     <div>
       {styling === "exercise" ? (
         <s.EditButton onClick={handleOpen}>
-          <img src="https://zeeguu.org/static/images/file_rename_orange_36dp.svg" alt="edit" />
+          <img
+            src="https://zeeguu.org/static/images/file_rename_orange_36dp.svg"
+            alt="edit"
+          />
         </s.EditButton>
       ) : styling === "match" ? (
         <sc.EditIconNoPadding onClick={handleOpen}>
-          <img src="https://zeeguu.org/static/images/file_rename_orange_36dp.svg" alt="edit" />
+          <img
+            src="https://zeeguu.org/static/images/file_rename_orange_36dp.svg"
```

Comment:
let's use www.zeeguu.org pls. currently we have nginx rewrite all variations by adding the www but in the future we might be more strict. 

also, did we not add a the domain as a constant or configuration option somewhere? This will become important if we want to test on a different domain, e.g. the staging environment on zeeguu.com.

Reply:
I have created a new file **appConstants.js** with the variable of the _PROD_ZEEGUU_URL_. Let me know if that's what you had in mind? 

This is not something I have changed, this was just the auto format that made these edits.

Reply:
Cool!